### ISSUE TITLE
Mean and Standard Deviation for the datasets while normalizing

### ISSUE THREAD

**Issue by meghbhalerao** [MedMNIST/MedMNIST#23]
Dear Authors,
Thank you for the dataset. 
I am looking at the getting_started.ipynb, for pathmnist it is said that the normalization transform is the following - `data_transform = transforms.Compose([
    transforms.ToTensor(),
    transforms.Normalize(mean=[.5], std=[.5])])
`
The values 0.5, 0.5 are being used. I have the following questions.
1. Does this value work for all the datasets in medmnist?
2. Is 0.5, 0.5 the correct mean and standard deviations, or are they just approximate numbers?
3. Is there a place where I can find datasets and their corresponding mean and standard deviation values so I can use them in my method?

Thanks for your time and help,
Megh

**Comment by duducheng** [comment]
Thanks for your questions ;)

The MedMNIST is a dataset, we don't consider how users will use it. The current normalization and any other pre-processing are only something we show "well, you can do like this", but you can use your own pre-processing anyway.

So, to answer your questions.
1. Yes, in our code, but you can also use your own.
2. No. They are something we specify by experience.
3. You can calculate them by yourself. The data is there, isn't it?

Jiancheng

**Comment by meghbhalerao** [comment]
Hi @duducheng,

Thanks a lot for your message. Got it!
Thanks for the clarification. Closing the issue. In case people need a quick script to find the mean and std, here it goes - 
[https://discuss.pytorch.org/t/about-normalization-using-pre-trained-vgg16-networks/23560/6?u=megh_bhalerao](https://discuss.pytorch.org/t/about-normalization-using-pre-trained-vgg16-networks/23560/6?u=megh_bhalerao
)

Megh
